backend/src/utils/githubUtil.ts

```diff
@@ -246,16 +228,32 @@ const validateReview = async (
     // comments made by user within the date range
     const eligibleComments = filterComments(review.comments, username, start, end);
 
-    // placeholder logic: valid if at least 10 words total
-    const totalWordCount = eligibleComments.reduce(
-      (count, comment) => count + comment.content.split(' ').length,
-      0
-    );
-    if (totalWordCount < 10) {
+    if (reviewIsTrivial(eligibleComments)) {
       throw new Error('Trivial review.');
     }
   });
 
+/** Returns `comments` with duplicate content from quote replies removed. */
+const removeQuoted = (comments: Comment[]): Comment[] => {
+  // quote replies are of the form "> (original comment)\r\n(new comment)"
+  const quotePattern = />(.*?)\r\n/gs;
+  return comments.map((comment) => ({
+    ...comment,
+    content: comment.content.replace(quotePattern, '').trim()
+  }));
+};
```

Comment:
This isolates the comment that was made by the user, so that the content that is being quoted doesnt count towards the requirement.